example-library/src/main/java/uk/gov/gchq/palisade/example/rule/BankDetailsRule.java

```diff
@@ -47,13 +49,10 @@ public Employee apply(final Employee record, final User user, final Context cont
         requireNonNull(user);
         requireNonNull(context);
         String purpose = context.getPurpose();
+        Set<String> roles = user.getRoles();
 
-        if (user instanceof ExampleUser) {
```

Comment:
How would this rule work with the PAYROLL role and the PAYROLL training course?

Reply:
It might be worth having the Role and Purpose if statement 1st and then an esleIf for the Payroll Training and Purpose case then an else to return the redacted record.

Reply:
Removed PAYROLL role in favour of PAYROLL_TRAINING_COURSE